DebugLogEnable.h

```diff
@@ -6,12 +6,16 @@
 #undef ASSERT
 #undef ASSERTM
 
+#ifndef LOG_PREAMBLE
+  #define LOG_PREAMBLE LOG_SHORT_FILENAME, __LINE__, __func__
+#endif
```

Comment:
[nit] After all this time, I prefer the default output to remain the same as before :) (I may change it myself later, please understand!)

Reply:
No problem, mind specifying exactly what you'd like it to look like?

Is this the what you mean with "Same as before"?
`[LEVEL] FILE L.# FUNC : message...`

Which would mean something like:
`#define LOG_PREAMBLE LOG_SHORT_FILENAME, "L.", __LINE__, " ", __func__, ":"`

Reply:
Done